### PR TITLE
Added "service" option.

### PR DESCRIPTION
**Pull request by j-bennet**
Using dsn-based approach (no config parsing required).
Connect #289.

### CODE REVIEW

File: pgcli/pgexecute.py [hunk 1]
```diff
@@ -175,6 +189,16 @@ def connect(self, database=None, user=None, password=None, host=None,
...
 
+    def _select_one(self, cur, sql):
+        """
+        Helper method to run a select and retrieve a single field value
+        :param cur: cursor
+        :param sql: string
+        :return: string
+        """
+        cur.execute(sql)
+        return cur.fetchall()[0][0]
```
Comment (amjith) [inline pgcli/pgexecute.py]: I'd prefer to use `cur.fetchone()` instead of `fetchall()`.

**Comment by amjith** [comment]
The PR works as expected. It's nice that we don't have to handle the parsing of pg_service.conf in pgcli. 

Good find that dsn is handled automatically by psycopg2. 

I think once you replace fetchall() with fetchone(), you might consider getting rid of the _select_one() function since it will be doing nothing more than proxying the function call.

**Comment by j-bennet** [comment]
Good point about fetchone! Changed that. I'm not sure I want to get rid of helper function, since it's not a 1-liner but 2-liner, and it's used 4 times.

**Comment by amjith** [comment]
Looks good. Thank you for handling this. :)

:100:

**Pull request merged**